package.json

```diff
@@ -11,7 +11,8 @@
     "storybook": "NODE_ENV=development start-storybook -p 8080 -c .storybook",
     "storybook:export": "build-storybook -c .storybook -o public",
     "speculate": "speculate",
-    "test": "jest --verbose"
+    "test": "jest --verbose",
+    "test:lint": "eslint --ext .js,jsx,json ./packages"
```

Comment:
Looks good for now. May need to add additional directories RE discussion in #19.